lib/api-wmata.js

```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+
+var props = require('../resources/properties');
+var secrets = require('../secret/secrets');
+
+var wmataReq = request.defaults({
+  baseUrl: props.wmata_base_url,
+  qs: secrets.apiKey
+});
+
+var WmataApi = function() {};
+
+WmataApi.prototype.get = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      console.error(endpoint.concat(': Error with WMATA'), error);
+      response.tell(props.wmataErrorSpeechOutput);
+    }
+  });
+};
+
+module.exports = new WmataApi();
```

Comment:
I don't think it's necessarily a best practice. If you did what I suggested, it's analogous to a Ruby Module. 

I think one thing a future refactor would entail is enabling passing the `apiKey` to the constructor. I would do that now so at least we're gaining some flexibility by turning it into an object.